arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -293,6 +296,9 @@ public List<Module> getModulesToCount() {
                 this.logData,
                 this.logInfo,
                 this.mod,
+                // TODO: not sure if this should be here, but if it should we have to define a limit
+                // for MODEXP in spillings.toml
+                // this.modexp.data(),
```

Comment:
We absolutely needs it here. The limit in `spillings.toml` is 0, the limits in the `*limits.toml` files is to be seen with Olivier and/or Alex.